tendrl/gluster_integration/objects/volume/atoms/expand/__init__.py

```diff
@@ -0,0 +1,89 @@
+import subprocess
```

Comment:
subprocess not used in this file.

Are you not running "tox -epep8" before submitting PR?